.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: ➕ Feature request
+about: Suggest an idea for this project
+title: '[FEATURE]'
+labels: 'Type/New Feature'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+<!-- A clear and concise description of what the problem is. Ex. I would like to have[...] -->
```

Comment:
A clear and concise description of what the problem is e.g., "I would like to have [...]"